platform/pulpcore/app/apps.py

```diff
@@ -17,6 +17,23 @@ def pulp_plugin_configs():
         if isinstance(app_config, PulpPluginAppConfig):
             yield app_config
 
+def pulp_plugin_config(plugin_app_label):
+    """
+    A getter of specific pulp plugin config
+
+    This makes it easy to retrieve a config for a specific Pulp plugin when looking for a
+    registered model.
+
+    Args:
+        plugin_label (basestring): Django app label of the pulp plugin
+
+    Returns:
+        :class:`pulpcore.app.apps.PulpPluginAppConfig`: The app config of the Pulp plugin.
+    """
+    for config in pulp_plugin_configs():
+        if config.label == plugin_app_label:
+            return config
+    raise RuntimeError("Plugin with Django app label %s is not installed." % plugin_app_label)
```

Comment:
Should this become a coded exception? Expecting to have something installed and it's not installed I think is something we want to raise in many places. What do you think?